frontend/mulighetsrommet-veileder-flate/src/components/tabell/TiltakstypeTabell.tsx

```diff
@@ -1,21 +1,37 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { Pagination, Table, Alert, Heading } from '@navikt/ds-react';
 import './Tabell.less';
-import { Tiltaksgjennomforing } from '../../../../mulighetsrommet-api-client';
 import Lenke from '../lenke/Lenke';
 import Kopiknapp from '../kopiknapp/Kopiknapp';
 import StatusGronn from '../../ikoner/Sirkel-gronn.png';
 import StatusGul from '../../ikoner/Sirkel-gul.png';
 import StatusRod from '../../ikoner/Sirkel-rod.png';
+import { sanityClient } from '../../sanityClient';
 
-export interface TiltakstypelisteProps {
-  tiltaksgjennomforingsliste: Array<Tiltaksgjennomforing>;
-}
-
-const TiltakstypeTabell = ({ tiltaksgjennomforingsliste }: TiltakstypelisteProps) => {
+const TiltakstypeTabell = () => {
   const [sort, setSort] = useState<any>();
   const [page, setPage] = useState(1);
   const rowsPerPage = 15;
+  const pagination = (tiltaksgjennomforing: string[]) => {
+    return Math.ceil(tiltaksgjennomforing.length / rowsPerPage);
+  };
+
+  const [tiltaksgjennomforing, setTiltaksgjennomforing] = useState([]);
+  const [tiltakstype, setTiltakstype] = useState([]);
+
+  useEffect(() => {
+    sanityClient
+      .fetch(`*[_type == "tiltaksgjennomforing"]`)
+      .then(data => setTiltaksgjennomforing(data))
+      .catch(console.error);
```

Comment:
Skulle vi brukt frontendlogger her for å gi beskjed i loggene om at vi ikke klarte å hente fra Sanity? Evt. vise en feilmelding til bruker?